server/common/default_config.yaml

```diff
@@ -0,0 +1,55 @@
+
+server:
+  verbose: true
```

Comment:
for most environments, the default verbose should be false

Reply:
Yes, agreed it should be false.